CIP-1694/README.md

```diff
@@ -1295,11 +1303,29 @@ The following table details the ratification requirements for each governance ac
 | 6. Treasury withdrawal                                            | ✓  | $P_6$    | \-       |
 | 7. Info                                                           | ✓  | $100$    | $100$    |
 
-Each of these thresholds is a governance parameter.
+Each of these thresholds is a governance parameter. There is one
+additional threshold, `Q5`, related to security relevant protocol
+parameters, which is explained below.
 The initial thresholds should be chosen by the Cardano community as a whole.
 The two thresholds for the Info action are set to 100% since setting it any lower
 would result in not being able to poll above the threshold.
 
+Some parameters are relevant to security properties of the system. Any
+proposal attempting to change such a parameter requires an additional
+vote of the SPOs, with the threshold `Q5`.
+
+The security relevant protocol parameters are:
+* `maxBBSize`
+* `maxTxSize`
+* `maxBHSize`
+* `maxValSize`
+* `maxBlockExUnits`
+* `minFeeA`
+* `minFeeB`
+* `coinsPerUTxOByte`
+* `govActionDeposit`
+* `minFeeRefScriptsCoinsPerByte`
+
```

Comment:
If this can be done at the protocol level, you should add `dvtMotionNoConfidence` and `pvtMotionNoConfidence` to this list as well and remove the CC rights to vote on these 2 parameters that were designed originaly to remove them.

They already cannot vote on "Motion of no confidence" and "Update Committee" governance actions, for obvious reasons. Which is why I think, their right to vote on these 2 parameters should be given to SPOs instead in my opinion. 😄👍

Reply:
Unfortunately that can't happen due to technical reasons, since `dvtMotionNoConfidence` and `dvtMotionNoConfidence` are not individual protocol parameters, but instead they are thresholds that are part of a threshold set, as such they can't be voted on individually. In other words, whenever a proposal is submitted that updates Stake Pool or DRep thresholds, all of the thresholds must be provided.

Moreover, drastic changes to the CIP are not possible at this stage.

The formal specification is complete and undergoing the audit at this point. Any changes right now will cause a significant delay to the release.

That being said, if you believe such functionality is very important, feel free to submit a CIP for the next era.

Reply:
Thank you for this explanation @lehins 
Would you happen to have more information on the mentioned "threshold set" and what they contain?
Are the sets the different governance action thresholds for DReps and SPOs?

Reply:
There are two separate protocol parameters one for [DReps](https://github.com/IntersectMBO/cardano-ledger/blob/e8f225c6a82ff02be63d5208fa72d11aee0e5999/eras/conway/impl/src/Cardano/Ledger/Conway/PParams.hs#L264-L274) and another for [Stake Pools](https://github.com/IntersectMBO/cardano-ledger/blob/e8f225c6a82ff02be63d5208fa72d11aee0e5999/eras/conway/impl/src/Cardano/Ledger/Conway/PParams.hs#L198-L204). Both of which contain all the relevant thresholds.

Reply:
> Thank you for this explanation @lehins 
> Would you happen to have more information on the mentioned "threshold set" and what they contain?
> Are the sets the different governance action thresholds for DReps and SPOs?